src/lib.rs

```diff
@@ -297,6 +297,8 @@ if_hyper! {
     pub mod cookie;
```

Comment:
Looks like this file and `src/error.rs` got their permissions modified?

Reply:
Sorry, fixed.